terraform/nowcasting/production/main.tf

```diff
@@ -4,12 +4,12 @@ Variables used across all modules
 locals {
   production_availability_zones = ["${var.region}a", "${var.region}b", "${var.region}c"]
   domain = "nowcasting"
+  modules_url = "github.com/openclimatefix/ocf-infrastructure//terraform/modules"
 }
 
 
 module "networking" {
-  source = "../../modules/networking"
-
+  source = "github.com/openclimatefix/ocf-infrastructure//terraform/modules/networking?ref=5b7005f"
```

Comment:
could we use the `modules_url` in as a variable in the source bit, to save duplication. 

Could we make a `5b7005f` variable then we can update it in variables without having to deploy new code?

Reply:
Sadly for the moment you can't put any interpolation at all in the source string, which a lot of people aren't all that pleased about! https://github.com/hashicorp/terraform/issues/1439

Reply:
ah shame, could you add that comment to the code, just so incase anyone else sees that

Reply:
Then, go for merge!!!